io/pci/PowerNVEEH.py.data/PowerNVEEH.yaml

```diff
@@ -0,0 +1,6 @@
+scenario:
+            #max_freeze: 5
```

Comment:
why all parameters are commented ?

Reply:
I remember from one of previous comments, that this is better, to avoid undesirable effect if any tester runs as is. So all are commented.

Reply:
Correct. But that can be achieved if the values to these parameters are left empty.

```
max_freeze: 5
function: 4
err:
pe:
```

Reply:
The problem with having these commented is, difficult to uncomment and change them automatically. Only changing can be done easily